libs/designsystem/src/lib/components/charts/chart-config/chart-stock-config/chart-stock-config.ts

```diff
@@ -0,0 +1,198 @@
+import {
+  BarController,
+  BarElement,
+  CategoryScale,
+  Chart,
+  ChartConfiguration,
+  ChartType,
+  ChartTypeRegistry,
+  Filler,
+  LinearScale,
+  LineController,
+  LineElement,
+  PointElement,
+  TimeScale,
+  Tooltip,
+  TooltipCallbacks,
+  TooltipItem,
+  TooltipLabelStyle,
+  TooltipOptions,
+} from 'chart.js';
+import annotationPlugin from 'chartjs-plugin-annotation';
+import { ColorHelper, DesignTokenHelper } from '../../../../helpers';
+import { ChartLocale } from '../../shared';
+import { ChartBaseConfig } from '../chart-base-config';
+
+import {
+  dataLabelsPluginConfig,
+  getDataLabelPosition,
+} from './data-label-position/data-label-position';
+import { getVerticalLinePluginConfig } from './vertical-line-plugin/vertical-line-plugin';
+
+const { getThemeColorHexString, getThemeColorRgbString } = ColorHelper;
+const { fontSize } = DesignTokenHelper;
+
+/**
+ * This class is still experimental. Feel free to use it, but please note that it is still subject to breaking changes.
+ */
+export class StockChartConfig extends ChartBaseConfig {
+  private static STOCK_CHART_LOCALE_DEFAULT: ChartLocale = 'en-US';
```

Comment:
This locale setting comes from the original stock chart, right? 
Do we know why we want it hardcoded, and not follow the platform?

Reply:
Maybe a bit of a shame if this is not made configurable to begin with, since the chart otherwise seems to accept different locales throughout the code.